neat_core.c

```diff
@@ -5247,6 +5243,7 @@ neat_connect(struct neat_he_candidate *candidate, uv_poll_cb callback_fx)
     const char *algo;
 #endif
     int enable = 1;
+    int disable = 0;
```

Comment:
@trozensztrauch `travis-ci` [fails](https://travis-ci.org/NEAT-project/neat/jobs/292060833) due to this unused variable, even though there is no git conflict. Could you please fix asap and PR?

Reply:
Thanks. I have just pushed a commit that should fix the problem. Let see if passes travis check now.